src/nvim/highlight_group.c

```diff
@@ -339,6 +339,16 @@ static const char *highlight_init_both[] = {
   "default link @lsp.type.typeParameter @type.definition",
   "default link @lsp.type.variable      @variable",
 
+  "default link @lsp.type.text          Pmenu",
```

Comment:
@mfussenegger  Several new highlight groups have been added here and temporarily placed in the lsp.type scope.. Any ideas?

Reply:
Note  that these are LSP semantic tokens, which are specified in the [specification](https://microsoft.github.io/language-server-protocol/specifications/lsp/3.17/specification/#textDocument_semanticTokens). We should _not_ invent new ones.

Reply:
Use different names (and a different scheme) for those, e.g., `PmenuText` etc.

Reply:
I  don't think `Pmenu` prefix is available,  these are two different things.  kind also in [spec](https://microsoft.github.io/language-server-protocol/specifications/lsp/3.17/specification/#completionItemKind)

Reply:
Then `CompletionKindText` etc. The point is that you shouldn't (ab)use the semantic token highlight groups for this.

Reply:
Or `@lsp.kind.text` would work too (depending on whether you can and want to reuse them for `SymbolKind`).

Reply:
And in any case, these need to be documented _somewhere_.

(Reminder to self: I need to add a consolidated list or "table of contents" in an appropriate place since it's getting hard for colorscheme authors to know which "default groups" they need to set.)

Reply:
`@lsp.kind` soud a good scope

Reply:
maybe also need `cmp` as a secondary prefix ? `lsp.cmp.kind.xx`..because there also have symbolkind https://microsoft.github.io/language-server-protocol/specifications/lsp/3.17/specification/#symbolKind

Reply:
Yes, that's what I was asking. Personally, since there is a large overlap between these and I don't see much valid reason for giving them different colors where they coincide, I would prefer to have a single `kind` table with everything that is _either_ a `CompletionItemKind` or a `SymbolKind` (or both).

Reply:
Oh, I ignored that sentence. .. becasue i was busy with other PRs before.. sry nvm